game.js

```diff
@@ -0,0 +1,230 @@
+    var x = e.code;
+    keys[x] = e.type == "keydown";
+    keyControl();
+    e.preventDefault;}
+,false)
+
+
+// [ADO] ked stlacis naraz 3 klavesy, tak sa nestane nic ?? pripadne vsetky kontroly dat do osobitnej metody napr isMovingUp(){
+//    if(keys['ArrowUp'] === true && keys['ArrowRight'] === false && keys['ArrowLeft'] === false && keys['ArrowDown'] === false){
+//      return true;
+//    }
+//    return false;
+//}
+// a potom by bola kontrola len if(isMovingUp){player.animation("up");} --> prehladnejsie
+function keyControl(){
+    if(keys['ArrowUp'] === true && keys['ArrowRight'] === false && keys['ArrowLeft'] === false && keys['ArrowDown'] === false){player1.animation('up');}
+    if(keys['ArrowUp'] === false && keys['ArrowRight'] === true && keys['ArrowLeft'] === false && keys['ArrowDown'] === false){player1.animation('right')}
+    if(keys['ArrowUp'] === false && keys['ArrowRight'] === false && keys['ArrowLeft'] === false && keys['ArrowDown'] === true){player1.animation('down')}
+    if(keys['ArrowUp'] === false && keys['ArrowRight'] === false && keys['ArrowLeft'] === true && keys['ArrowDown'] === false){player1.animation('left')}
+
+    if(keys['ArrowUp'] === true && keys['ArrowRight'] === false && keys['ArrowLeft'] === true && keys['ArrowDown'] === false){player1.animation('upleft');}
+    if(keys['ArrowUp'] === true && keys['ArrowRight'] === true && keys['ArrowLeft'] === false && keys['ArrowDown'] === false){player1.animation('upright');}
+    if(keys['ArrowUp'] === false && keys['ArrowRight'] === false && keys['ArrowLeft'] === true && keys['ArrowDown'] === true){player1.animation('downleft');}
+    if(keys['ArrowUp'] === false && keys['ArrowRight'] === true && keys['ArrowLeft'] === false && keys['ArrowDown'] === true){player1.animation('downright');}
+    
+    if(keys['Space'] === true){
+        if (gameBar.ammo > 1) {
+                player1.sx = 0; 
+                player1.sy = 344.4; 
+                player1.sw = 857; 
+                player1.w = 227;
+                playerShoots.push(new Shoot()); gameBar.ammo -= 1;
+                setTimeout(() => {
+                player1.sx = 770; 
+                player1.sy = 0; 
+                player1.sw = 759.8 ; 
+                player1.w = 200;}, 60);
+                }
+        else gameBar.ammo = 'NO AMMO';
+    }
+
+    if(keys['ArrowUp'] === false && keys['ArrowRight'] === false && keys['ArrowLeft'] === false && keys['ArrowDown'] === false){player1.intervalStop()}
+    if(keys['Space'] === false){
+                player1.sx = 0;    
+                player1.sy = 0; 
+                player1.sw = 759.8; 
+                player1.w = 200;}
+}
+
+// Collision among two objects
+
+//[ADO] tu by som dal skor ten nazov hasCollision, a ono to hlada asi 2 objekty ktore sa nejako stretnu, avsak hlada list...co sa stane v pripade, ak je viac
+//objektov na sebe ?? potom pouzivas len jeden index stale nizsie v metode 
+function collision(object1 = [] , object2 = []){
```

Comment:
zmenit nazov...a co sa stane ak nastane viac konfliktov ?? vrati to len jeden alebo ako to momentalne funguje ?? ak budem mat nejake objecty hned nad sebou a budu kolidovat s hracom naraz, tak vieme to ohandlovat ??